persistence/sql/persister_oauth2.go

```diff
@@ -353,8 +437,40 @@ func (p *Persister) DeletePKCERequestSession(ctx context.Context, signature stri
 	return p.deleteSessionBySignature(ctx, signature, sqlTablePKCE)
 }
 
-func (p *Persister) RevokeRefreshToken(ctx context.Context, id string) error {
-	return p.deactivateSessionByRequestID(ctx, id, sqlTableRefresh)
+func (p *Persister) RevokeRefreshToken(ctx context.Context, requestId string) error {
+	return p.deactivateSessionByRequestID(ctx, requestId, sqlTableRefresh)
+}
+
+func (p *Persister) RevokeRefreshTokenMaybeGracePeriod(ctx context.Context, requestId string, signature string) error {
```

Comment:
Please add tests for this new functionality

Reply:
I can't figure out how to write a test in `fosite_store_helpers.go` that changes the configuration value that this depends on. Can you point me in the right direction?

Reply:
Hm yeah that looks a bit iffy. I think you can call `store.Config().MustSet(key, value)` but you most likely need to reset it to the original value on `t.Cleanup` so other tests don't fail.

Reply:
@aeneasr I used reflection to update the config, and enabled the test that exercises this function